bin/sortedstats_cacher.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import configparser
+from datetime import datetime, timedelta
+from zoneinfo import ZoneInfo
```

Comment:
NAV still doesn't require newer Pythons than 3.7, and `zoneinfo` was introduced to the stdlib first in 3.9 - so I assume this will fail spectacularly once the CI suite tests on 3.7 and 3.8.

Maybe @hmpf has some tips on how to handle this in a portable fashion?

Reply:
Solved this using a backport package